tests/Testcontainers.Tests/Fixtures/Containers/Unix/DockerTlsFixture.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DotNet.Testcontainers.Tests.Fixtures
+{
+  using DotNet.Testcontainers.Builders;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+
+  [UsedImplicitly]
+  public sealed class DockerTlsFixture : ProtectDockerDaemonSocket
```

Comment:
Based on the Docker [docs](https://docs.docker.com/engine/security/protect-access/#other-modes) I would assume `DockerTlsFixture` should use following configuration: `tls`, `tlscert`, `tlskey`. The same applies here as well as for `DockerTlsFixture`. I am not sure if `--tlsverify=false` corresponds to that configuration.

Reply:
No it does not corresponds to docker configuration. It corresponds to image used to run run dockerd.